src/components/GridTemplates/DetailsPage1/DetailsPage1.stories.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file Details page template #1 story.
+ * @copyright IBM Security 2020
+ */
+
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { disableCentered, patterns } from '../../../../.storybook';
+import {
+  theme,
+  DetailsPage1,
+  DetailsPage1Main,
+  DetailsPage1Section,
+  DetailsPage1SectionTerm,
+  DetailsPage1SectionDefinition,
+  DetailsPage1Aside,
+} from '../../..';
+
+function DemoContent() {
+  return (
+    <div
+      style={{
+        backgroundColor: theme.interactive04,
+        minHeight: '240px',
+        width: '100%',
+      }}
+    >
+      lorem ipsum
+    </div>
+  );
+}
+
+disableCentered(storiesOf(patterns('DetailsPage1'), module)).add(
```

Comment:
Would you mind using [Component Story Format](https://storybook.js.org/docs/formats/component-story-format) here? I believe the `storiesOf` format will be deprecated in the future.

Reply:
Where is this being used elsewhere in the codebase?

Reply:
This is currently in place for - 

- [`TimeIndicator`](https://github.com/carbon-design-system/ibm-security/blob/master/src/components/TimeIndicator/TimeIndicator.stories.js)
- [`TrendingCard`](https://github.com/carbon-design-system/ibm-security/blob/master/src/components/TrendingCard/TrendingCard.stories.js)
- [`UNSTABLE__Shell`](https://github.com/carbon-design-system/ibm-security/blob/next/src/components/UNSTABLE__Shell/Shell.stories.js)

As per the [Storybook team](https://storybook.js.org/docs/formats/component-story-format/), this is now the recommended way of writing stories - in my opinion, we should be writing new stories in this format moving forward, to avoid a larger migration effort in future.

Reply:
@SimonFinney how should `disableCentered` be applied in this branch?

Reply:
Good question! I believe we have support for this in `next` - we may need to bring the change over

Reply:
So in that case, I wouldn't be able to use the component story format without an interesting workaround unfortunately. This component is full-width and not centered.

Reply:
Let's leave it for now... I'll create a stub to revisit and bring in the configuration from `next`